### ISSUE TITLE
Uncaught InvalidArgumentException: "" is not a valid region

### ISSUE THREAD

**Issue by gregdotca** [inverse/tuya-client-php#4]
Is this project currently functional, or not quite yet? I'm getting the below error when trying to run the example code (after filling in my own details).

````
[21-Nov-2018 10:59:48 Canada/Pacific] PHP Fatal error:  Uncaught InvalidArgumentException: "" is not a valid region in X:\dev\sandbox\vendor\webmozart\assert\src\Assert.php:1099
Stack trace:
#0 X:\dev\sandbox\vendor\webmozart\assert\src\Assert.php(553): Webmozart\Assert\Assert::reportInvalidArgument('"" is not a val...')
#1 X:\dev\sandbox\vendor\inverse\tuya-client\src\Region.php(23): Webmozart\Assert\Assert::oneOf('', Array, '%s is not a val...')
#2 X:\dev\sandbox\vendor\inverse\tuya-client\src\Region.php(43): Inverse\TuyaClient\Region->__construct('')
#3 X:\dev\sandbox\vendor\inverse\tuya-client\src\ApiClient.php(69): Inverse\TuyaClient\Region::fromAccessToken(Object(Inverse\TuyaClient\Token))
#4 X:\dev\sandbox\vendor\inverse\tuya-client\src\ApiClient.php(79): Inverse\TuyaClient\ApiClient->checkAccessToken()
#5 X:\dev\sandbox\vendor\inverse\tuya-client\src\ApiClient.php(47): Inverse\TuyaClient\ApiClient->request('Discovery', 'discovery')
#6 X:\dev\sandbox\index.php(21): Inverse\TuyaClient\ApiClient->discoverDevices()
#7 {main}
  thrown in X:\dev\sandbox\vendor\webmozart\assert\src\Assert.php on line 1099
````

**Comment by inverse** [comment]
@chetcuti sorry for the delay in replying. This project is very WIP.

I've improved the error messaging displayed so it should provide the information for the mapper that'll need to be updated.

**Comment by inverse** [comment]
Fixed by defaulting to US as per the official py lib.

Released `0.2.1-alpha`
